packages/attribute-registry-process/src/routers/AttributeRouter.ts

```diff
@@ -105,6 +105,7 @@ const attributeRouter = (
             SUPPORT_ROLE,
             SECURITY_ROLE,
             M2M_ROLE,
+            M2M_ADMIN_ROLE,
```

Comment:
The authorization test in `tests/api/getAttributeByOriginAndCode.test.ts` should be updated